sage-ios/SAGE/SAGE/School.swift

```diff
@@ -16,6 +16,8 @@ class School: NSObject, NSCoding {
     var director: User?
     var address: String?
     
+    var studentCount: Int = -1
```

Comment:
make this 0. only IDs should default to -1, int properties should also default to 0 if it makes more sense, like totalHours as well